src/django_ixctl/rest/urls/ixctl.py

```diff
@@ -1,8 +1,48 @@
 from django.urls import path, include
-
-import django_ixctl.rest.views.ixctl
+from django_ixctl.rest.views.ixctl import InternetExchange, Members, Routeserver
 import django_ixctl.rest.route.ixctl
 
+ix_list = InternetExchange.as_view({
+    'get': 'list',
+    'post': 'create'
+})
+
+ix_detail = InternetExchange.as_view({
+    'get': 'retrieve',
+})
+
+ix_import_peeringdb = InternetExchange.as_view({
+    'post': 'import_peeringdb',
+})
+
+member_list = Members.as_view({
+    'get': 'list',
+    'post': 'create'
+})
+
+member_detail = Members.as_view({
+    'put': 'update',
+    'delete': 'destroy',
+})
+
+routeserver_list = Routeserver.as_view({
+    'get': 'list',
+    'post': 'create',
+})
+
+routeserver_detail = Routeserver.as_view({
+    'put': 'update',
+    'delete': 'destroy',
+})
+
 urlpatterns = [
-    path("", include(django_ixctl.rest.route.ixctl.router.urls)),
+    path('ix/<str:org_tag>/', ix_list, name='ix-list'),
+    path('ix/<str:org_tag>/import_peeringdb', ix_import_peeringdb, name='ix-import-peeringdb'),
+    path('ix/<str:org_tag>/<str:ix_tag>/', ix_detail, name='ix-detail'),
+    path('member/<str:org_tag>/<str:ix_tag>/', member_list, name='member-list'),
+    path('member/<str:org_tag>/<str:ix_tag>/<int:member_id>', member_detail, name='member-detail'),
+    path('routeserver/<str:org_tag>/<str:ix_tag>/', routeserver_list, name='routeserver-list'),
```

Comment:
should be `rs` instead of `routeserver` - also lets use the reftag set on the view class for this

Reply:
Ah okay good thought!

Reply:
Addressed in https://github.com/fullctl/ixctl/pull/35/commits/e2bbf46ff5416d7fc07d1ee0cb227197e38a7ef6